lib/state.js

```diff
@@ -242,13 +243,18 @@ StateResolver.prototype.isPathInNodeModulesDirectory_ = function(path) {
 };
 
 
-StateResolver.prototype.resolveFrame_ = function(frame) {
-  var args = this.resolveArgumentList_(frame);
+StateResolver.prototype.resolveFrame_ = function(frame, resolveVars) {
+  var args = [];
```

Comment:
It might be more succinct to write var args = resolveVars ? this.resolveArgumentList_(frame) : [];